outbound.js

```diff
@@ -1492,7 +1514,12 @@ HMailItem.prototype.try_deliver_host_on_socket = function (mx, host, port, socke
         return send_command('MAIL', 'FROM:' + self.todo.mail_from);
     };
 
+    var fp_called = false;
```

Comment:
I worry about this value getting toggled to true and then remaining true forever. Does outbound spawn and destroy instances, or run and run and run?